altos-rust/port/cortex-m0/src/peripheral/usart/defs.rs

```diff
@@ -1,9 +1,33 @@
-// Base addresses for USART 1 and 2
+/*
+ * Copyright © 2017 AltOS-Rust Team
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+ */
+
+//! This file contains the constants associated with the bit definitions
+//! for the registers being used.
+//! This is not a complete listing, however, all constants used throughout
+//! the program are listed here, there are bit definitions that are listed
+//! and not being used, or not listed at all.
+
+/// Base addresses for USART 1 and 2
 pub const USART1_ADDR: *const u32 = 0x4001_3800 as *const _;
 pub const USART2_ADDR: *const u32 = 0x4000_4400 as *const _;
 
 // ------------------------------------
-// USARTx - CR1 Bit definitions
+/// USARTx - CR1 Bit definitions
```

Comment:
Nope all the way down (though these won't actually do anything since none of these constants are public facing)